src/main/java/org/apache/maven/plugins/dependency/exclusion/AnalyzeExclusionsMojo.java

```diff
@@ -57,15 +58,19 @@
 @Mojo(name = "analyze-exclusions", requiresDependencyCollection = ResolutionScope.TEST, threadSafe = true)
 public class AnalyzeExclusionsMojo extends AbstractMojo {
 
-    @Component
     private MavenProject project;
```

Comment:
Constructor injected fields should be made `final`.

Reply:
good idea, done